CMakeModules/Dependencies.cmake

```diff
@@ -12,7 +22,42 @@ endif()
 function(PREBUILT_DEPENDENCIES_RESOLVE targetName)
 endfunction()
 
-if(PREBUILT_DEPENDENCIES)
+if(BUILD_PLATFORM STREQUAL "android")
+    add_library(ffmpeg::avcodec SHARED IMPORTED)
+    set_target_properties(ffmpeg::avcodec PROPERTIES
+        INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_SOURCE_DIR}/FFmpeg/android/${ANDROID_ABI}/include
```

Comment:
That's because ffmpeg doesn't support cmake so it can't be included directly to build system and is builded externally. Also CMake finders are expecting to find things in projects it currently builds or in sysroot. As ffmpeg artifacts are neither build by CMake nor reside in sysroot and instead produced by custom script they are placed by it in the specific place. And all this code are just dummies to give CMake directions to find things.

Now, why CMAKE_SOURCE_DIR, if you look [there](
https://github.com/OpenEnroth/OpenEnroth_Android/tree/master/openenroth/jni) you will see that structure here is that OpenEnroth is just one of subprojects, so CMAKE_SOURCE_DIR in this case is pointing to `jni` directory. And custom script is placing everything in `jni/FFMpeg/android/<architecture>/`.

Reply:
Got it. That's a non-trivial cross-repository dependency, so let's at least add a TODO to straighten it out.

Or we can simply add a cmake option here, and set it in https://github.com/OpenEnroth/OpenEnroth_Android/blob/master/openenroth/jni/CMakeLists.txt.